MAUI/Rating/Appearance-Customization.md

```diff
@@ -0,0 +1,213 @@
+    <rating:SfRating.RatingSettings>
+        <rating:RatingSettings RatedFill="Red"/>
+    </rating:SfRating.RatingSettings>
+</rating:SfRating>
+	
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfRating rating= new SfRating();
+rating.Value = 3;
+RatingSettings ratingSettings = new RatingSettings();
+ratingSettings.RatedFill = Colors.Red;
+rating.RatingSettings = ratingSettings;
+           
+{% endhighlight %}
+
+{% endtabs %}
+
+![Rated item fill color](images/rated-fill.png)
+
+### Unrated Items
+
+The [`UnratedFill`] property fills the unrated item with the specified solid color in the [`SfRating`].
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<rating:SfRating Value="3">
+    <rating:SfRating.RatingSettings>
+        <rating:RatingSettings UnratedFill="Gray"/>
+    </rating:SfRating.RatingSettings>
+</rating:SfRating>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfRating rating= new SfRating();
+rating.Value = 3;
+RatingSettings ratingSettings = new RatingSettings();
+ratingSettings.UnratedFill = Colors.Gray;
+rating.RatingSettings = ratingSettings;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Unrated item fill color](images/unrated-fill.png)
+
+## Set Stroke
+
+[`SfRating`] control has support to set the stroke color for the rated and unrated items.
+
+### Rated Items
+
+The [`RatedStroke`] property sets the stroke for the rated item with the specified solid color for the rated items in the [`SfRating`] control.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<rating:SfRating Value="3">
+    <rating:SfRating.RatingSettings>
+        <rating:RatingSettings RatedStroke="Black"/>
+    </rating:SfRating.RatingSettings>
+</rating:SfRating>
+	
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfRating rating= new SfRating();
+rating.Value = 3;
+RatingSettings ratingSettings = new RatingSettings();
+ratingSettings.RatedStroke = Colors.Black;
+rating.RatingSettings = ratingSettings;	
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Rated item stroke](images/rated-stroke.png)
+
+### Unrated Items
+
+The [`UnratedStroke`] property sets the stroke for the unrated area with the specified solid color in the [`SfRating`] control.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<rating:SfRating Value="3">
+    <rating:SfRating.RatingSettings>
+        <rating:RatingSettings UnratedStroke="Black"/>
+    </rating:SfRating.RatingSettings>
+</rating:SfRating>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfRating rating= new SfRating();
+rating.Value = 3;
+RatingSettings ratingSettings = new RatingSettings();
+ratingSettings.UnratedStroke = Colors.Black;
+rating.RatingSettings = ratingSettings;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Unrated item stroke](images/unrated-stroke.png)
+ 
+## Set Stroke Thickness
+
+[`SfRating`] control has support to set the stroke thickness for the rated and unrated items.
+
+### Rated Items
+
+The [`RatedStrokeThickness`] property sets the stroke thickness for the rated item with the specified value in the [`SfRating`] control.
```

Comment:
The [`RatedStrokeThickness`] property sets the stroke thickness for the rated items with the specified value in the [`SfRating`] control.

Reply:
Changed